modules/core/src/lib/layer.ts

```diff
@@ -755,10 +754,11 @@ export default abstract class Layer<PropsT extends {} = {}> extends Component<
     }
 
     if (bufferLayoutChanged) {
+      // AttributeManager is always defined when this method is called
       const attributeManager = this.getAttributeManager();
```

Comment:
Can we add the exclamation mark here instead?
```suggestion
      const attributeManager = this.getAttributeManager()!;
```